server/controllers/apply/applications/pagesController.test.ts

```diff
@@ -148,6 +148,20 @@ describe('pagesController', () => {
       )
     })
 
+    it('redirects to the tasklist if there is no next page', async () => {
+      page.next.mockReturnValue(undefined)
```

Comment:
Incredibly minor but would setting `page.next` to `undefined` more accurately describe the behaviour? 
EG `page.next = undefined`
What we have here is fine if the types don't like it

Reply:
Yeah, it didn't like it. Will keep it as is